src/shared/user-record-show.c

```diff
@@ -124,13 +124,25 @@ void user_record_show(UserRecord *hr, bool show_full_group_info) {
         case -ESTALE:
                 printf(" Password OK: %slast password change in future%s\n", ansi_highlight_yellow(), ansi_normal());
                 break;
-
         default:
                 if (r < 0) {
                         errno = -r;
                         printf(" Password OK: %sno%s (%m)\n", ansi_highlight_yellow(), ansi_normal());
                         break;
                 }
+                if (!hr->hashed_password) {
```

Comment:
please use: 

```c
if (strv_isempty(hr->hashed_password))
```

i.e. we allow for string arrays to ways that they can be empty: if the pointer to the array is NULL or if the first element of the array is NULL. strv_isempty() hides that distinction, returns true for both cases.